en/SUMMARY.md

```diff
@@ -133,6 +133,8 @@
 - [Graphics Rendering](module-map/graphics.md)
     - [Render Pipeline](render-pipeline/overview.md)
         - [Built-in Render Pipeline](render-pipeline/builtin-pipeline.md)
+    - [Graphics Backend](graphics-backend/overview.md)
+        - [WebGPU](graphics-backend/WebGPU.md)
```

Comment:
Please do not add this doc, instead add webgpu build information into web platform documentation